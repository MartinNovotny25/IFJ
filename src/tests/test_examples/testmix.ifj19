a=1

def foo(a, b, c, d):
    """ random
    pomocna funkce
    pro cokoliv, bere
    parametr a,b,c,d"""

    print(a)
    e = b+c-d*b/d//d

    print(e)

while a+1>(1+3*a+4):
    print(a)

foo('', 0, 1)

#random radkova funkce
""" random vice radkova funkce na jednom radku"""

""" random vice radkova
funkce na vice "" "" "" \f \t \n
radcich """

foo("""string
pro funkci""", 1, 2, 4)
