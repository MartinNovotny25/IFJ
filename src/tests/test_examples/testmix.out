TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(-)
TOKEN(IDENTIFIKATOR)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(/)
TOKEN(IDENTIFIKATOR)
TOKEN(//)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 1)
TOKEN(>)
TOKEN(()
TOKEN(INTEGER, 1)
TOKEN(+)
TOKEN(INTEGER, 3)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 4)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, )
TOKEN(,)
TOKEN(INTEGER, 0)
TOKEN(,)
TOKEN(INTEGER, 1)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, string
pro funkci)
TOKEN(,)
TOKEN(INTEGER, 1)
TOKEN(,)
TOKEN(INTEGER, 2)
TOKEN(,)
TOKEN(INTEGER, 4)
TOKEN())
TOKEN(EOL)
TOKEN(EOF)
